code/modules/mob/living/silicon/robot/robot_damage.dm

```diff
@@ -1,11 +1,18 @@
-/mob/living/silicon/robot/updatehealth()
+mob/living/silicon/robot/updatehealth()

```

Comment:
would I sound nitpicky to point out that this should actually have a `/` before it
`/mob/living/silicon/robot/updatehealth()`